libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1469,28 +1469,29 @@ Json::Value StandardCompiler::compileYul(InputsAndSettings _inputsAndSettings)
 	if (runtimeObject.bytecode)
 		runtimeObject.bytecode->link(_inputsAndSettings.libraries);
 
-	for (string const& objectKind: vector<string>{"bytecode", "deployedBytecode"})
+	struct ObjectKind { string const kind; bool isByteCode; };
+	for (auto const& objectKind: vector<ObjectKind>{{"bytecode", true}, {"deployedBytecode", false}})
```

Comment:
```suggestion
	for (auto&& [kind, isByteCode]: {make_pair("bytecode", true), make_pair("deployedBytecode", false)})
```
That should work, shouldn't it :-D? If we want to be fanciful at least :-).

Reply:
Fun quiz question: what's the type of the expression after the colon in that suggestion :-)...

Reply:
By the way - it may be nicer to invert the ``bool`` and call it ``runtimeObject``.

Reply:
I don't see the argument for the inverting, it is checked for both, true and false at different places ;)
I do like the make_pair solution though. I suppose it is a pair again?

Reply:
ah, it seems to be a tuple?

Reply:
I think the entire bracketed expression will be a ``std::initializer_list<std::pair<const char*, bool>>`` in the end - but I'd also need to recheck to be sure :-).

Reply:
About inverting and renaming the flag: the main reason for the suggestion is that both the ``bytecode`` and the ``deployedBytecode``... well are byte code, so ``isByteCode`` is a weird name. And the most natural choice I found for renaming is the name of the argument of ``collectEVMObject``, which is ``runtimeObject`` (but is inverted)...

Reply:
Or ``isRuntimeObject``, otherwise it conflicts with the ``runtimeObject`` we already have :-).